Mathlib/Algebra/Order/Hom/Normed.lean

```diff
@@ -0,0 +1,77 @@
+/-
+Copyright (c) 2024 Yakov Pechersky. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yakov Pechersky
+-/
+import Mathlib.Algebra.Order.Hom.Basic
+import Mathlib.Analysis.Normed.Group.Basic
+
+/-!
+# Constructing (semi)normed groups from (semi)normed homs
+
+This file defines constructions that upgrade `(Comm)Group` to `(Semi)Normed(Comm)Group`
+using a `Group(Semi)normClass` when the codomain is the reals.
+
+See `Mathlib.Algebra.Order.Hom.Ultra` for further upgrades in to nonarchimedean normed groups.
```

Comment:
```suggestion
See `Mathlib.Algebra.Order.Hom.Ultra` for further upgrades in to nonarchimedean normed groups.
```
See `Mathlib.Algebra.Order.Hom.Ultra` for further upgrades to nonarchimedean normed groups.